apps/web/src/api/api.client.ts

```diff
@@ -161,6 +161,14 @@ export function buildApiHttpClient({
       });
     },
 
+    async getPreferences(workflowId: string) {
+      return get('/v1/preferences?workflowId=' + workflowId);
```

Comment:
Prefer to use template literals:

```suggestion
      return get(`/v1/preferences?workflowId=${workflowId}`);
```